internal/pkg/task/env_runner.go

```diff
@@ -60,7 +62,7 @@ func (r *EnvRunner) Run() ([]*Task, error) {
 		return nil, fmt.Errorf(fmtErrPublicSubnetsFromEnv, r.Env, err)
 	}
 	if len(subnets) == 0 {
-		return nil, errNoSubnetFound
+		return nil, fmt.Errorf(fmtErrNoSubnetFoundInEnv, r.Env)
```

Comment:
Instead of having the error message contain a recommendation "(need %s and %s tags?)" can we use the `log` pkg to add additional context? So the error can be specific but logging can suggest additional actions.

```suggestion
		log.Warningf("Subnets need to be tagged with %s and %s. Are your subnets tagged?\n", deploy.AppTagKey, deploy.EnvTag.Key)
		return nil, errNoSubnetFound
```